src/main/scala/main.scala

```diff
@@ -7,9 +7,23 @@ import org.scalajs.dom.document
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import java.time.temporal.WeekFields
+import kaptenallocweb.timeEdit.TimeEdit
 
 @main def run: Unit = 
-    document.addEventListener("DOMContentLoaded", (e: dom.Event) => setupUI())  
+    document.addEventListener("DOMContentLoaded", (e: dom.Event) => 
+      TimeEdit.fetchData(
+        onLoad = xhr => {
+          if xhr.status == 200 then
+            val diffs = TimeEdit.findDiscrepancies(timeEditData = xhr.responseText, kaptenAllocData = dataGeneratedFromKaptenAlloc)
+            if diffs.nonEmpty then addDiscrepancyPanel(diffs)
```

Comment:
suggestion: add `else ()` on this line to make it complete and this non-dangling...